packages/framework/src/Resources/translations/messages.cs.po

```diff
@@ -2296,6 +2296,9 @@ msgstr "Počet objednávek"
 msgid "Number of products in filter"
 msgstr "Počty zboží ve filtru"
 
+msgid "Number of selected products is not limited, however only 30 will be displayed on frontend."
```

Comment:
Shouldn't there be a variable here that is replaced by a constant from the code where the item limit is specified?